tensorrt_llm/inputs/registry.py

```
@@ -99,7 +105,7 @@ def __call__(
                     inputs["prompt"],
                     add_special_tokens=sampling_params.add_special_tokens,
                     **kwargs)
-            except:
+            except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)